### ISSUE TITLE
Linux Mint Debian Edition  5 signal 11

### ISSUE THREAD

**Issue by RahRah80** [zaps166/NFSIISE#93]
Need For Speed II SE
  Wrapper v1.3.0
  Game    v1.1.2
  OpenGL  1
Application closed with a signal: 11
Segmentation fault (core dumped)

**Comment by zaps166** [comment]
Do you have 32-bit drivers and required libs installed? Do you use correct game files (NFSIISE)?

**Comment by RahRah80** [comment]
Hello.

Yes i have 32-bit.

I copied the game files from Special Edition and files/directories are all
in lower case.
The  game shows the videos and closes.

**Comment by zaps166** [comment]
Game is working for me (Manjaro Linux, Radeon GPU). Could you compile in debug and run via GDB and show backtrace?

**Comment by RahRah80** [comment]
Ok, but how do i do it?

**Comment by zaps166** [comment]
```bash
git clone https://github.com/zaps166/NFSIISE
git submodule update --init --recursive
./compile_nfs debug
cd "Need For Speed II SE"
gdb nfs2se
```

in `GDB` start by pressing "r" and enter. Before starting copy/symlink `gamedata` and `fedata` into `Need For Speed II SE`. Don't forget about lower case (`convert_to_lowercase` script).
When it crashes, write `bt` and enter (in GDB).

**Comment by RahRah80** [comment]
GNU gdb (Debian 10.1-1.7) 10.1.90.20210103-git
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html
>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from nfs2se...
(gdb) r
Starting program: /home/rah-rah/git/NFSIISE/Need For Speed II SE/nfs2se
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Need For Speed II SE
  Wrapper v1.3.0
  Game    v1.1.2
  OpenGL  1
[New Thread 0xf6d68b40 (LWP 10108)]
[New Thread 0xee1eeb40 (LWP 10111)]
[New Thread 0xed6beb40 (LWP 10112)]
[New Thread 0xeccbdb40 (LWP 10113)]
[New Thread 0xec4bcb40 (LWP 10114)]
[New Thread 0xeb8ffb40 (LWP 10115)]
[New Thread 0xeb0feb40 (LWP 10116)]
[New Thread 0xea6ffb40 (LWP 10117)]
[New Thread 0xe9cffb40 (LWP 10118)]
[New Thread 0xe92ffb40 (LWP 10119)]
[New Thread 0xe88ffb40 (LWP 10120)]
[New Thread 0xe80feb40 (LWP 10121)]
[New Thread 0xe74ffb40 (LWP 10122)]
[New Thread 0xe6cfeb40 (LWP 10123)]
[New Thread 0xe62ffb40 (LWP 10124)]
[New Thread 0xe58feb40 (LWP 10125)]
[New Thread 0xe4effb40 (LWP 10126)]
[Thread 0xe4effb40 (LWP 10126) exited]
[New Thread 0xe4effb40 (LWP 10129)]
[Thread 0xe4effb40 (LWP 10129) exited]
[New Thread 0xed988b40 (LWP 10131)]
[New Thread 0xe4effb40 (LWP 10133)]
[New Thread 0xd36feb40 (LWP 10134)]
[Thread 0xd36feb40 (LWP 10134) exited]
[New Thread 0xd36feb40 (LWP 10136)]
[New Thread 0xd2efdb40 (LWP 10138)]
[New Thread 0xd26fcb40 (LWP 10139)]
[New Thread 0xd1cffb40 (LWP 10141)]
[New Thread 0xc12ffb40 (LWP 10144)]
[New Thread 0xc0afeb40 (LWP 10145)]
[New Thread 0xc02fdb40 (LWP 10146)]
[New Thread 0xbfafcb40 (LWP 10147)]
[Thread 0xc02fdb40 (LWP 10146) exited]
[New Thread 0xc02fdb40 (LWP 10148)]
[Thread 0xc02fdb40 (LWP 10148) exited]
[New Thread 0xc02fdb40 (LWP 10152)]

Thread 1 "nfs2se" received signal SIGSEGV, Segmentation fault.
sub_487060 () at Asm/NFS2SE.asm:177499
177499 mov ecx, [eax+1Ch]
(gdb)

The before compiling i didn't do "git submodule update --init --recursive"

**Comment by zaps166** [comment]
Are you able to try on Windows?

**Comment by RahRah80** [comment]
i don't have windows installed, i can use wine.
i already installed what needed to compile for windows, however i have no
idea what is "path/to/lib/folder" and "path/to/include/folder"
/usr/lib
/usr/include
are theses correct?

**Comment by RahRah80** [comment]
After making the changes i got theses errors when compiling

Building OpenGL 1
Building release for Windows... In file included from
/usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_joystick.h:42,
                 from DInput.h:30,
                 from DInput.c:25:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_audio.h:31,
                 from EAcsnd.c:45:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from Glide2x/OpenGL1.c:27,
                 from Glide2x.c:28:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/share/mingw-w64/include/winsock.h:32,
                 from /usr/share/mingw-w64/include/windows.h:92,
                 from Kernel32.h:31,
                 from Kernel32.c:25:
/usr/share/mingw-w64/include/_timeval.h:10:8: error: redefinition of
‘struct timeval’
   10 | struct timeval
      |        ^~~~~~~
In file included from /usr/include/sys/select.h:37,
                 from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Kernel32.h:28,
                 from Kernel32.c:25:
/usr/include/bits/types/struct_timeval.h:8:8: note: originally defined here
    8 | struct timeval
      |        ^~~~~~~
In file included from /usr/share/mingw-w64/include/winsock.h:36,
                 from /usr/share/mingw-w64/include/windows.h:92,
                 from Kernel32.h:31,
                 from Kernel32.c:25:
/usr/share/mingw-w64/include/psdk_inc/_fd_types.h:25:3: error: conflicting
types for ‘fd_set’
   25 | } fd_set;
      |   ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Kernel32.h:28,
                 from Kernel32.c:25:
/usr/include/sys/select.h:70:5: note: previous declaration of ‘fd_set’ was
here
   70 |   } fd_set;
      |     ^~~~~~
In file included from /usr/share/mingw-w64/include/windows.h:92,
                 from Kernel32.h:31,
                 from Kernel32.c:25:
/usr/share/mingw-w64/include/winsock.h:299:34: error: conflicting types for
‘select’
  299 |   WINSOCK_API_LINKAGE int WSAAPI select(int nfds,fd_set
*readfds,fd_set *writefds,fd_set *exceptfds,const PTIMEVAL timeout);
      |                                  ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Kernel32.h:28,
                 from Kernel32.c:25:
/usr/include/sys/select.h:101:12: note: previous declaration of ‘select’
was here
  101 | extern int select (int __nfds, fd_set *__restrict __readfds,
      |            ^~~~~~
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_timer.h:31,
                 from Kernel32.c:27:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_timer.h:31,
                 from Timer.c:27:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_messagebox.h:25,
                 from User32.c:27:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/share/mingw-w64/include/winsock.h:32,
                 from /usr/share/mingw-w64/include/windows.h:92,
                 from WinMM.c:28:
/usr/share/mingw-w64/include/_timeval.h:10:8: error: redefinition of
‘struct timeval’
   10 | struct timeval
      |        ^~~~~~~
In file included from /usr/include/sys/select.h:37,
                 from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from WinMM.c:25:
/usr/include/bits/types/struct_timeval.h:8:8: note: originally defined here
    8 | struct timeval
      |        ^~~~~~~
In file included from /usr/share/mingw-w64/include/winsock.h:36,
                 from /usr/share/mingw-w64/include/windows.h:92,
                 from WinMM.c:28:
/usr/share/mingw-w64/include/psdk_inc/_fd_types.h:25:3: error: conflicting
types for ‘fd_set’
   25 | } fd_set;
      |   ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from WinMM.c:25:
/usr/include/sys/select.h:70:5: note: previous declaration of ‘fd_set’ was
here
   70 |   } fd_set;
      |     ^~~~~~
In file included from /usr/share/mingw-w64/include/windows.h:92,
                 from WinMM.c:28:
/usr/share/mingw-w64/include/winsock.h:299:34: error: conflicting types for
‘select’
  299 |   WINSOCK_API_LINKAGE int WSAAPI select(int nfds,fd_set
*readfds,fd_set *writefds,fd_set *exceptfds,const PTIMEVAL timeout);
      |                                  ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from WinMM.c:25:
/usr/include/sys/select.h:101:12: note: previous declaration of ‘select’
was here
  101 | extern int select (int __nfds, fd_set *__restrict __readfds,
      |            ^~~~~~
In file included from /usr/include/SDL2/SDL_stdinc.h:31,
                 from /usr/include/SDL2/SDL_main.h:25,
                 from /usr/include/SDL2/SDL.h:32,
                 from Wrapper.c:27:
/usr/include/SDL2/SDL_config.h:4:10: fatal error: SDL2/_real_SDL_config.h:
No such file or directory
    4 | #include <SDL2/_real_SDL_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/share/mingw-w64/include/winsock2.h:52,
                 from Wsock32.h:56,
                 from Wsock32.c:25:
/usr/share/mingw-w64/include/_timeval.h:10:8: error: redefinition of
‘struct timeval’
   10 | struct timeval
      |        ^~~~~~~
In file included from /usr/include/sys/select.h:37,
                 from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Wsock32.h:28,
                 from Wsock32.c:25:
/usr/include/bits/types/struct_timeval.h:8:8: note: originally defined here
    8 | struct timeval
      |        ^~~~~~~
In file included from /usr/share/mingw-w64/include/winsock2.h:56,
                 from Wsock32.h:56,
                 from Wsock32.c:25:
/usr/share/mingw-w64/include/psdk_inc/_fd_types.h:25:3: error: conflicting
types for ‘fd_set’
   25 | } fd_set;
      |   ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Wsock32.h:28,
                 from Wsock32.c:25:
/usr/include/sys/select.h:70:5: note: previous declaration of ‘fd_set’ was
here
   70 |   } fd_set;
      |     ^~~~~~
In file included from Wsock32.h:56,
                 from Wsock32.c:25:
/usr/share/mingw-w64/include/winsock2.h:1025:34: error: conflicting types
for ‘select’
 1025 |   WINSOCK_API_LINKAGE int WSAAPI select(int nfds,fd_set
*readfds,fd_set *writefds,fd_set *exceptfds,const PTIMEVAL timeout);
      |                                  ^~~~~~
In file included from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:394,
                 from Wrapper.h:33,
                 from Wsock32.h:28,
                 from Wsock32.c:25:
/usr/include/sys/select.h:101:12: note: previous declaration of ‘select’
was here
  101 | extern int select (int __nfds, fd_set *__restrict __readfds,
      |            ^~~~~~

**Comment by zaps166** [comment]
try binaries from my repo first for Windows and Linux

**Comment by RahRah80** [comment]
Hello.

Search all over github and can't find the binary files

**Comment by RahRah80** [comment]
Hello.

Using your binaries the result is the same.

On Linux i get exit signal 11.

With the Windows version using Wine i got:

Need For Speed II SE
  Wrapper v1.3.0
  Game    v1.1.2
  OpenGL  1
0010:err:ole:marshal_object couldn't get IPSFactory buffer for interface
{00000131-0000-0000-c000-000000000046}
0010:err:ole:marshal_object couldn't get IPSFactory buffer for interface
{6d5140c1-7436-11ce-8034-00aa006009fa}
0010:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub,
hres=0x80004002
0010:err:ole:CoMarshalInterface Failed to marshal the interface
{6d5140c1-7436-11ce-8034-00aa006009fa}, 80004002
0010:err:ole:get_local_server_stream Failed: 80004002
0009:err:ole:CoGetClassObject class {bcde0395-e52f-467c-8e3d-c4579291692e}
not registered
0009:err:ole:CoGetClassObject no class object
{bcde0395-e52f-467c-8e3d-c4579291692e} could be created for context 0x1
0009:err:ole:CoGetClassObject class {bcde0395-e52f-467c-8e3d-c4579291692e}
not registered
0009:err:ole:CoGetClassObject no class object
{bcde0395-e52f-467c-8e3d-c4579291692e} could be created for context 0x1
0009:err:dsound:get_mmdevenum CoCreateInstance failed: 80040154
0009:err:ole:CoGetClassObject class {bcde0395-e52f-467c-8e3d-c4579291692e}
not registered
0009:err:ole:CoGetClassObject no class object
{bcde0395-e52f-467c-8e3d-c4579291692e} could be created for context 0x1
0009:err:dsound:get_mmdevenum CoCreateInstance failed: 80040154
0009:err:ole:CoGetClassObject class {25e609e4-b259-11cf-bfc7-444553540000}
not registered
0009:err:ole:CoGetClassObject no class object
{25e609e4-b259-11cf-bfc7-444553540000} could be created for context 0x1
0009:err:ole:CoUninitialize Mismatched CoUninitialize
0009:err:ole:CoGetClassObject class {25e609e4-b259-11cf-bfc7-444553540000}
not registered
0009:err:ole:CoGetClassObject no class object
{25e609e4-b259-11cf-bfc7-444553540000} could be created for context 0x1
0009:err:ole:CoUninitialize Mismatched CoUninitialize
SDL_Init failed: Haptic error CoCreateInstance
0013:err:ole:CoGetClassObject class {529a9e6b-6587-4f23-ab9e-9c7d683e3c50}
not registered
0013:err:ole:CoGetClassObject no class object
{529a9e6b-6587-4f23-ab9e-9c7d683e3c50} could be created for context 0x1
0013:err:ole:CoGetClassObject class {529a9e6b-6587-4f23-ab9e-9c7d683e3c50}
not registered
0013:err:ole:CoGetClassObject no class object
{529a9e6b-6587-4f23-ab9e-9c7d683e3c50} could be created for context 0x1
0009:err:ntdll:RtlpWaitForCriticalSection section 0x7be9e8e0 "virtual.c:
csVirtual" wait timed out in thread 0009, blocked by 0019, retrying (60 sec)
0013:err:ntdll:RtlpWaitForCriticalSection section 0x7b0094 "heap.c:
HEAP.critSection" wait timed out in thread 0013, blocked by 0009, retrying
(60 sec)
Killed


Also tested with PlayOnLinux and crashed

Unhandled exception: page fault on read access to 0x0000001c in 32-bit code
(0x00481763).
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:006b GS:0063
 EIP:00481763 ESP:0032fc28 EBP:00000010 EFLAGS:00210246(  R- --  I  Z- -P- )
 EAX:00000000 EBX:004aab32 ECX:0118be26 EDX:00000000
 ESI:0071acc0 EDI:00000001
Stack dump:
0x0032fc28:  00000001 0071acc0 0032fc40 0118be26
0x0032fc38:  004aab32 00408a13 65665c2e 61746164
0x0032fc48:  5c63705c 74786574 6977735c 33317373
0x0032fc58:  6e66662e 00000000 00000001 00000081
0x0032fc68:  f7c7ba37 0032fd10 0048000d 0118bf58
0x0032fc78:  0032fca8 7bc573c0 000000f0 007cac70
Backtrace:
=>0 0x00481763 EntryPoint+0xffffffff() in nfs2se (0x00000010)
0x00481763 EntryPoint+0xffffffff in nfs2se: movl 0x1c(%eax),%ecx
Modules:
Module Address Debug info Name (152 modules)
PE  400000-  7ac000 Dwarf           nfs2se
PE 6c940000-6cabf000 Deferred        sdl2
ELF 72d70000-745a5000 Deferred        libz3.so.4
ELF 745a8000-79053000 Deferred        libllvm-11.so.1
ELF 79058000-7a7fa000 Deferred        radeonsi_dri.so
ELF 7a800000-7a943000 Deferred        opengl32<elf>
  \-PE 7a840000-7a943000 \               opengl32
ELF 7b000000-7b0e2000 Deferred        kernelbase<elf>
  \-PE 7b020000-7b0e2000 \               kernelbase
ELF 7b400000-7b673000 Deferred        kernel32<elf>
  \-PE 7b420000-7b673000 \               kernel32
ELF 7bc00000-7beb3000 Deferred        ntdll<elf>
  \-PE 7bc30000-7beb3000 \               ntdll
ELF 7c000000-7c005000 Deferred        <wine-loader>
ELF 7c908000-7c944000 Deferred        msctf<elf>
  \-PE 7c910000-7c944000 \               msctf
ELF 7caa0000-7cac9000 Deferred        libtinfo.so.6
ELF 7cad0000-7cb08000 Deferred        libedit.so.2
ELF 7cb08000-7cb12000 Deferred        libffi.so.7
ELF 7cb18000-7cb22000 Deferred        libatomic.so.1
ELF 7cb28000-7cb47000 Deferred        libgcc_s.so.1
ELF 7cd18000-7cd7a000 Deferred        libvulkan.so.1
ELF 7cd80000-7cd8a000 Deferred        libdrm_nouveau.so.2
ELF 7cd90000-7cd9d000 Deferred        libdrm_amdgpu.so.1
ELF 7cda0000-7cdbe000 Deferred        libelf.so.1
ELF 7cdc0000-7cdcf000 Deferred        libdrm_radeon.so.1
ELF 7cdd0000-7cde1000 Deferred        libsensors.so.5
ELF 7cde8000-7cdf2000 Deferred        libxcb-xfixes.so.0
ELF 7cdf8000-7ce01000 Deferred        libxcb-sync.so.1
ELF 7ce08000-7ce0d000 Deferred        libxcb-present.so.0
ELF 7ce10000-7ce2e000 Deferred        libxcb-glx.so.0
ELF 7ce30000-7cead000 Deferred        libglx_mesa.so.0
ELF 7ceb0000-7ceec000 Deferred        libglx.so.0
ELF 7cef0000-7cf6d000 Deferred        libgldispatch.so.0
ELF 7cfc8000-7cfcb000 Deferred        libxshmfence.so.1
ELF 7cfd0000-7cfd6000 Deferred        libxcb-dri3.so.0
ELF 7cfd8000-7cfdd000 Deferred        libxcb-shm.so.0
ELF 7cfe0000-7cfe7000 Deferred        libxcb-dri2.so.0
ELF 7cfe8000-7cffe000 Deferred        libdrm.so.2
ELF 7d100000-7d129000 Deferred        libglapi.so.0
ELF 7d130000-7d19d000 Deferred        libgl.so.1
ELF 7d1d0000-7d1e9000 Deferred        xinput1_4<elf>
  \-PE 7d1e0000-7d1e9000 \               xinput1_4
ELF 7d1f0000-7d22c000 Deferred        uxtheme<elf>
  \-PE 7d200000-7d22c000 \               uxtheme
ELF 7d230000-7d27f000 Deferred        usp10<elf>
  \-PE 7d240000-7d27f000 \               usp10
ELF 7d280000-7d414000 Deferred        comctl32<elf>
  \-PE 7d2b0000-7d414000 \               comctl32
ELF 7d418000-7d47e000 Deferred        dinput8<elf>
  \-PE 7d430000-7d47e000 \               dinput8
ELF 7d480000-7d49b000 Deferred        hid<elf>
  \-PE 7d490000-7d49b000 \               hid
ELF 7d4a0000-7d4a8000 Deferred        libkeyutils.so.1
ELF 7d4a8000-7d4b8000 Deferred        libkrb5support.so.0
ELF 7d4b8000-7d4eb000 Deferred        libk5crypto.so.3
ELF 7d4f0000-7d5cc000 Deferred        libkrb5.so.3
ELF 7d5d0000-7d62a000 Deferred        libgssapi_krb5.so.2
ELF 7d630000-7d665000 Deferred        libtirpc.so.3
ELF 7d668000-7d690000 Deferred        libgpg-error.so.0
ELF 7d690000-7d6aa000 Deferred        libresolv.so.2
ELF 7d6b0000-7d6bc000 Deferred        libogg.so.0
ELF 7d6c0000-7d71d000 Deferred        libopus.so.0
ELF 7d720000-7d7ad000 Deferred        libvorbisenc.so.2
ELF 7d7b0000-7d7dc000 Deferred        libvorbis.so.0
ELF 7d7e0000-7d81b000 Deferred        libflac.so.8
ELF 7d820000-7d83b000 Deferred        libnsl.so.2
ELF 7d840000-7d927000 Deferred        libgcrypt.so.20
ELF 7d928000-7d94c000 Deferred        liblz4.so.1
ELF 7d950000-7da21000 Deferred        libzstd.so.1
ELF 7da28000-7da54000 Deferred        liblzma.so.5
ELF 7da58000-7daf0000 Deferred        libsndfile.so.1
ELF 7daf0000-7dbb0000 Deferred        libsystemd.so.0
ELF 7dbb0000-7dc12000 Deferred        libdbus-1.so.3
ELF 7dc18000-7dca6000 Deferred        libpulsecommon-14.2.so
ELF 7dca8000-7dd06000 Deferred        libpulse.so.0
ELF 7dd68000-7dd6d000 Deferred        libcom_err.so.2
ELF 7dd70000-7dd7c000 Deferred        librt.so.1
ELF 7dd80000-7dd88000 Deferred        libasyncns.so.0
ELF 7dd88000-7dd94000 Deferred        libwrap.so.0
ELF 7dd98000-7dd9d000 Deferred        libxdamage.so.1
ELF 7dda0000-7dda5000 Deferred        libx11-xcb.so.1
ELF 7dda8000-7ddbe000 Deferred        avrt<elf>
  \-PE 7ddb0000-7ddbe000 \               avrt
ELF 7ddc8000-7ddfc000 Deferred        winepulse<elf>
  \-PE 7ddd0000-7ddfc000 \               winepulse
ELF 7de00000-7de28000 Deferred        mmdevapi<elf>
  \-PE 7de10000-7de28000 \               mmdevapi
ELF 7de50000-7de58000 Deferred        libxfixes.so.3
ELF 7de58000-7de65000 Deferred        libxcursor.so.1
ELF 7de68000-7de7c000 Deferred        libxi.so.6
ELF 7de80000-7de85000 Deferred        libxcomposite.so.1
ELF 7de88000-7de95000 Deferred        libxrandr.so.2
ELF 7de98000-7dea4000 Deferred        libxrender.so.1
ELF 7dea8000-7deaf000 Deferred        libxxf86vm.so.1
ELF 7deb0000-7deb5000 Deferred        libxinerama.so.1
ELF 7deb8000-7dec7000 Deferred        libmd.so.0
ELF 7dec8000-7dee0000 Deferred        libbsd.so.0
ELF 7dee0000-7dee7000 Deferred        libxdmcp.so.6
ELF 7dee8000-7deed000 Deferred        libxau.so.6
ELF 7def0000-7df1e000 Deferred        libxcb.so.1
ELF 7df20000-7e072000 Deferred        libx11.so.6
ELF 7e078000-7e08e000 Deferred        libxext.so.6
ELF 7e0c0000-7e17d000 Deferred        winex11<elf>
  \-PE 7e0e0000-7e17d000 \               winex11
ELF 7e340000-7e34a000 Deferred        libuuid.so.1
ELF 7e350000-7e37e000 Deferred        libexpat.so.1
ELF 7e380000-7e3cd000 Deferred        libfontconfig.so.1
ELF 7e3d0000-7e3f3000 Deferred        libbrotlicommon.so.1
ELF 7e3f8000-7e406000 Deferred        libbrotlidec.so.1
ELF 7e408000-7e426000 Deferred        libz.so.1
ELF 7e428000-7e468000 Deferred        libpng16.so.16
ELF 7e468000-7e52f000 Deferred        libfreetype.so.6
ELF 7e560000-7e59d000 Deferred        ws2_32<elf>
  \-PE 7e570000-7e59d000 \               ws2_32
ELF 7e5a0000-7e5d0000 Deferred        msacm32<elf>
  \-PE 7e5b0000-7e5d0000 \               msacm32
ELF 7e5d0000-7e69a000 Deferred        winmm<elf>
  \-PE 7e5e0000-7e69a000 \               winmm
ELF 7e6a0000-7e6c7000 Deferred        shcore<elf>
  \-PE 7e6b0000-7e6c7000 \               shcore
ELF 7e6c8000-7e72b000 Deferred        shlwapi<elf>
  \-PE 7e6e0000-7e72b000 \               shlwapi
ELF 7e730000-7f124000 Deferred        shell32<elf>
  \-PE 7e760000-7f124000 \               shell32
ELF 7f128000-7f1aa000 Deferred        setupapi<elf>
  \-PE 7f140000-7f1aa000 \               setupapi
ELF 7f1b0000-7f2f0000 Deferred        oleaut32<elf>
  \-PE 7f1e0000-7f2f0000 \               oleaut32
ELF 7f2f0000-7f389000 Deferred        rpcrt4<elf>
  \-PE 7f310000-7f389000 \               rpcrt4
ELF 7f390000-7f500000 Deferred        ole32<elf>
  \-PE 7f3c0000-7f500000 \               ole32
ELF 7f500000-7f5d6000 Deferred        msvcrt<elf>
  \-PE 7f530000-7f5d6000 \               msvcrt
ELF 7f5d8000-7f5ef000 Deferred        version<elf>
  \-PE 7f5e0000-7f5ef000 \               version
ELF 7f5f0000-7f82f000 Deferred        user32<elf>
  \-PE 7f620000-7f82f000 \               user32
ELF 7f830000-7f98e000 Deferred        gdi32<elf>
  \-PE 7f850000-7f98e000 \               gdi32
ELF 7f990000-7fa14000 Deferred        advapi32<elf>
  \-PE 7f9a0000-7fa14000 \               advapi32
ELF 7fe90000-7fea5000 Deferred        libnss_files.so.2
ELF 7fea8000-7ffac000 Deferred        libm.so.6
ELF 7ffb8000-7ffdf000 Deferred        imm32<elf>
  \-PE 7ffc0000-7ffdf000 \               imm32
ELF f7b78000-f7b7e000 Deferred        libdl.so.2
ELF f7b80000-f7d69000 Deferred        libc.so.6
ELF f7d70000-f7d92000 Deferred        libpthread.so.0
ELF f7dc8000-f7f7c000 Dwarf           libwine.so.1
ELF f7f80000-f7fac000 Deferred        ld-linux.so.2
Threads:
process  tid      prio (all id:s are in hex)
00000008 nfs2se.exe
00000009    0
0000002e    0
0000002f    0
00000030    0
00000033    0
00000035    1
0000003a    2
0000003b    2
0000003c    2
0000003d    2
0000003f    1
0000000e services.exe
0000000f    0
00000010    0
00000013    0
0000001a    0
00000021    0
00000011 plugplay.exe
00000012    0
00000016    0
00000017    0
00000018 winedevice.exe
00000019    0
0000001b    0
0000001c    0
0000001d    0
0000001e    0
0000001f winedevice.exe
00000020    0
00000022    0
00000023    0
00000024    0
00000025 (D) C:\Program Files\nfs2sew\nfs2se.exe
00000026    0 <==
00000034    0
00000036    0
00000037    0
00000038    0
00000039    1
00000040    2
00000041    2
00000042    2
00000043    2
00000045    1
00000047    2
00000027 explorer.exe
00000028    0
0000002b    0
0000002c    0
0000002d    0
System information:
    Wine build: wine-5.0.3 (Debian 5.0.3-3)
    Platform: i386
    Version: Windows 7
    Host system: Linux
    Host version: 5.10.0-19-amd64

**Comment by charsi** [comment]
same error for me on linux mint -
```
$ ./nfs2se
Need For Speed II SE
  Wrapper v1.3.0
  Game    v1.1.2
  OpenGL  1
Application closed with a signal: 11
Segmentation fault (core dumped)
```

**Comment by zaps166** [comment]
On Manjaro and Arch and real Windows it works. If it crashes on real Windows, it means game data causes a problem.

**Comment by RahRah80** [comment]
ok, i'll look into it.

thanks
